gitpod-io/gitpod
# [installer] Integrate cloud sql proxy

## Description
This PR integrates the cloud sql proxy into the installer.

## Related Issue(s)

Fixes #6562

## How to test
1. Enable `CloudSQL` in the installer:
```YAML
database:
  cloudSQL:
    connectionName: "sje-self-hosted-playground:europe-west1:gitpod-foobar"
    serviceAccount:
      kind: "secret"
      name: "gcp-sa"
```

2. Ensure your secret is in place:
```
installer validate cluster --config config.yaml
```

3. Install gitpod using the installer

## Release Notes

```release-note
NONE
```
